uefi-services/Cargo.toml

```diff
@@ -7,8 +7,14 @@ edition = "2018"
 
 [dependencies]
 rlibc = "1"
+x86_64 = "0.2"
 
 uefi = { path = ".." }
 uefi-alloc = { path = "../uefi-alloc" }
 uefi-logger = { path = "../uefi-logger" }
 log = { version = "0.4", default-features = false }
+
+[features]
+# Signals the implementation that QEMU's exit port hack is enabled and assigned
+# to the f4 CPU port.
+qemu-f4-exit = []
```

Comment:
Just a minor nitpick: it seems your editor might be removing the [trailing newlines](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline), and Git (and possibly other UNIX tools) might not like it.

Reply:
(it's still missing here, ded0656 fixed only the other file)